src/OutputMaker.js

```diff
@@ -0,0 +1,49 @@
+const { INPUT_CHAR } = require('./Constants/InputValues');
+const { STATUS } = require('./Constants/BridgeStatus');
+
+/**
+ * 이동에 따른 출력을 생성하는 역할을 한다.
+ */
+const OutputMaker = {
+  /**
+   * 생성된 진행상황 배열을 반환하는 메서드
+   */
+  makeOutput(input, pass) {
+    const upAndDownArray = this.upAndDown(input, pass);
+    const splitUpAndDownArray = this.splitUpAndDown(upAndDownArray);
+    const result = [
+      [splitUpAndDownArray[0].join().replace(/,/gi, STATUS.seperator)],
+      [splitUpAndDownArray[1].join().replace(/,/gi, STATUS.seperator)],
+    ];
+    return result;
+  },
```

Comment:
```suggestion
  makeOutput(input, pass) {
    const upAndDownArray = this.upAndDown(input, pass);
    const [upBridge, downBridge ] = this.splitUpAndDown(upAndDownArray);
    const result = [
      [upBridge.join().replace(/,/gi, STATUS.seperator)],
      [downBridge.join().replace(/,/gi, STATUS.seperator)],
    ];
    return result;
  },
```

```suggestion
   makeOutput(input, pass) {
    const upAndDownArray = this.upAndDown(input, pass);
    const [upBridge, downBridge ] = this.splitUpAndDown(upAndDownArray);
    const result = [
      [replaceCommaWithSeperator(join(upBridge))],
      [replaceCommaWithSeperator(join(downBridge))],
    ];
    return result;
  },

  join(value) {
    return value.join();
  },

  replaceCommaWithSeperator(value) {
    return value.replace(/,/gi, STATUS.seperator)
  }
```
이 코드에서  splitUpAndDownArray[0] or [1]  이 부분이 처음 봤을 때 무엇을 뜻하는지 파악하기 조금 어려웠던 것 같습니다. 
밑에 내려가서 예상한게 맞나 확인 후 다시 봐야하기때문에  조금 다르게 표현하신다면  괜찮지 않을까? 라고 조심스럽게 생각해봅니다.

다른 표현 방식으로는 여러가지 방법이 있겠지만 저라면 어떻게 했을까? 생각하다가 두 번째 처럼 한 번 해보았습니다.